website/pages/docs/install.mdx

```diff
@@ -5,9 +5,15 @@ description: 'How to install Million.js into your React project'
 
 import { Callout, Tabs, Tab, Steps } from 'nextra-theme-docs';
```

Comment:
I can't do some comments on the manual side but similar comments for that

Reply:
Yeah, I already caught it, thanks!